lab-aaron/README.md

```diff
@@ -0,0 +1,6 @@
+This app can test 5 different methods:
```

Comment:
As you keep working through the labs, make sure your readme has a good explanation of what your code is doing. For example, a brief overview of how you used call, bind, and apply